gigadb/app/tools/files-metadata-console/tests/functional/UpdateFileSizeCest.php

```diff
@@ -51,8 +51,17 @@ public function tryUpdateFileSizeWhenContentLengthInBytes(\FunctionalTester $I):
     {
         $webClient = new Client([ 'allow_redirects' => false ]);
         $us = new URLsService();
-        $dfu = new DatasetFilesUpdater(["doi" => "100005", "us" => $us, "webClient" => $webClient]);
+        $dfu = new DatasetFilesUpdater(["doi" => "100142", "us" => $us, "webClient" => $webClient]);
         $success = $dfu->updateFileSize();
         $I->assertEquals(1, $success, "Not all files were updated successfully");
     }
+
+    public function tryUpdateFileSizesFromTsvFile(\FunctionalTester $I): void
+    {
+        $webClient = new Client([ 'allow_redirects' => false ]);
+        $us = new URLsService();
+        $dfu = new DatasetFilesUpdater(["doi" => "100142", "us" => $us, "webClient" => $webClient]);
+        $success = $dfu->updateFileSizes();
+//        $I->assertEquals(1, $success, "Not all files were updated successfully");
+    }
```

Comment:
**question:** This test is still WIP?

Reply:
I agree. This test has been removed now.